client/src/components/Layout/Layout.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import Header from "../Header/Header";
+import ArticleList from "../ArticleList/ArticleList";
+
+const mockedArticles = [
+    {id: 1, title: 'Eduard helps me in Frontend', text: 'I like it because............', date: '2023-02-04', },
+    {id: 2, title: 'Vitali part of this cool project', text: 'I like it because............', date: '2022-06-29', },
+]
+
+const Layout = () => {
```

Comment:
If your function doesn't have a body, the curvy braces and return can be omitted.
It's not a rule, just a notice.

```suggestion
const Layout = () => (
    <>
    ,,,
    </>
);
```